test_file.txt

```diff
@@ -0,0 +1 @@
+added test text
```

Comment:
So we'll need to delete this test file first. There are two ways to do so:

1. On your local environment, do `git rm test_file.txt`, followed by `git commit -m "Remove test_file.txt"`, and then `git push`.
2. Alternatively, go to https://github.com/srijac/pygmt/blob/add-alias-rose/test_file.txt, and there should be a delete button :wastebasket: that will remove the file. You will need to also enter a commit message and click on "Commit changes"